client/signup/config/flows.js

```diff
@@ -112,6 +112,21 @@ function getDomainSignupFlowDestination( { domainItem, cartItem, siteId, designT
 	return getThankYouNoSiteDestination();
 }
 
+function getEmailSignupFlowDestination( dependencies, localeSlug ) {
+	if ( dependencies.cartItem ) {
+		return getSignupDestination( dependencies, localeSlug );
+	}
+
+	if ( dependencies.domainItem ) {
+		return addQueryArgs(
+			{ siteId: dependencies.siteId },
+			`/checkout/thank-you/features/email-license/${ dependencies.siteSlug }/:receiptId`
```

Comment:
@sirbrillig I opted to use this slug pattern of `/checkout/thank-you/features/:feature` as defined [here](https://github.com/Automattic/wp-calypso/blob/6dd27ca99211505d5a15e561c5341c35ffbb4c21/client/my-sites/checkout/index.js#L142). 

I wanted to check with you that we aren't abusing a slug that's intended for something else?

Reply:
Thanks for asking! I don't know much about the `features` post-checkout pages (most post-checkout behavior has been added by other teams over the years), but if it works for you, I don't think it's a problem.

Reply:
Great, thanks for clarifying! Sounds like we should be good to use it if there are no known red flags.